src/sidebar/components/share-annotations-panel.js

```diff
@@ -0,0 +1,175 @@
+  const shareURI = ((frame, group) => {
+    if (!frame || !group) {
+      return '#';
+    }
+    return `https://hyp.is/go?url=${encodeURIComponent(mainFrame.uri)}&group=${
+      group.id
+    }`;
+  })(mainFrame, focusedGroup);
+
+  const trackShareClick = shareTarget => {
+    analytics.track(analytics.events.DOCUMENT_SHARED, shareTarget);
+  };
+
+  const copyShareLink = () => {
+    try {
+      copyText(shareURI);
+      flash.info('Copied share link to clipboard');
+    } catch (err) {
+      flash.error('Unable to copy link');
+    }
+  };
+
+  const inputRef = useRef();
+
+  const focusShareLink = () => {
+    inputRef.current.select();
+  };
+
+  return (
+    <SidebarPanel
+      title={panelTitle}
+      panelName={uiConstants.PANEL_SHARE_ANNOTATIONS}
+    >
+      {shouldRenderPanelContent && (
+        <div className="share-annotations-panel">
+          <div className="share-annotations-panel__intro">
+            {focusedGroup.type === 'private' ? (
+              <p>
+                Use this link to share these annotations with other group
+                members:
+              </p>
+            ) : (
+              <p>Use this link to share these annotations with anyone:</p>
+            )}
+          </div>
+          <div className="share-annotations-panel__input">
+            <input
+              ref={inputRef}
+              className="form-input share-annotations-panel__form-input"
+              type="text"
+              value={shareURI}
+              onClick={focusShareLink}
+              onFocus={focusShareLink}
+              readOnly
+            />
+            <button
+              onClick={copyShareLink}
+              title="copy share link"
+              aria-label="copy share link"
+              className="btn btn-clean share-annotations-panel__copy-btn"
+            >
+              <SvgIcon
+                name="copy"
+                className="share-annotations-panel__copy-btn-icon"
+              />
+            </button>
+          </div>
+          <p>
+            {focusedGroup.type === 'private' ? (
+              <span>
+                Annotations in the private group <em>{focusedGroup.name}</em>{' '}
+                are only visible to group members.
+              </span>
+            ) : (
+              <span>
+                Anyone using this link may view the annotations in the group{' '}
+                <em>{focusedGroup.name}</em>.
+              </span>
+            )}{' '}
+            <span>
+              Private (
+              <SvgIcon
+                name="lock"
+                inline
+                className="share-annotations-panel__icon--inline"
+              />{' '}
+              <em>Only Me</em>) annotations are only visible to you.
+            </span>
+          </p>
+          <div className="share-annotations-panel__links">
+            <div className="share-annotations-panel__link">
```

Comment:
personally like using `ul` `li` in these cases, but I can go either way

Reply:
Oh, I _like_ this. Gonna change it.